model/general/createCalendar.js

```diff
@@ -2,6 +2,36 @@ const client = require('../../database/pg.js');
 const utils = require('../../lib/hashUtils.js');
 
 const createCalendar = (data, callback) => {
+  const cookie = [ data.cookies.jobsite ];
```

Comment:
Not a real problem, but is there any reason you put spaces within the brackets? Usually that's more used for destructuring